### DISCUSSION TITLE
on configure.batrunning in Windows PC

### DISCUSSION THREAD

**Discussion by Subham2372** [toppers/users-forum#124] [Q&A]
Hello Mori,
Good Morning!!

as per our discussion previously - 
I have a question about the file a-comstack/can/target/hsbrh850f1k_ccrh/sample/configure.bat. 
for this you have mentioned that 
-----
If you try to run this bat file, you need cfg.exe is required.
https://www.toppers.jp/download.cgi/cfg-mingw-static-1_9_6.zip (for Windows)
---
But when i click the cfg.exe its open the terminal and close. Nothing happen. Do you except to install something out of here. ?

Thank you in advance!!!

**Comment by mitsut** [comment]
@Subham2372 
atk2, a-comstack, a-rtegen, cfg files are required.
and these folders structure as follows;
you can try to run configure.bat(a-constack)
```
.
├── a-comstack
│   ├── app
│   ├── can
│   │   ├── arch
│   │   ├── coverage
│   │   ├── doc
│   │   ├── stub
│   │   └── target
│   │       ├── hsbrh850f1k_ccrh
│   │       │   └── sample         <=== configure.bat is here.
│   │       ├── hsbrh850f1k_gcc
│   │       ├── hsbrh850f1l_ccrh
│   │       ├── hsbv850e2fg4_cx
│   │       └── nios2_dev_gcc
│   ├── can_fd
│   ├── canif
│   ├── cantp
│   ├── com
│   ├── doc
│   ├── general
│   ├── pdur
│   ├── stub
│   └── utils
│       └── abrex
├── a-rtegen
│   ├── bin
│   ├── doc
│   ├── ecu_extractor
│   ├── sample
│   └── src
├── atk2-sc1
│   ├── OBJ
│   ├── arch
│   ├── cfg
│   │   └── cfg    <=== cfg.exe file is here.
│   ├── doc
│   ├── include
│   ├── kernel
│   ├── library
│   ├── sample
│   ├── sysmod
│   ├── target
│   │   ├── hsbrh850f1k_ccrh
│   │   └── hsbrh850f1k_gcc
│   └── utils
```
